src/json/ProgramRequest.schema.json

```diff
@@ -21,7 +21,8 @@
       "oneOf": [
         { "$ref": "#/definitions/programTrigger" },
         { "$ref": "#/definitions/programValidation" },
-        { "$ref": "#/definitions/programIntrospection" }
+        { "$ref": "#/definitions/programIntrospection" },
+        { "$ref": "#/definitions/programTriggerVariablesSchemaRequest" }
```

Comment:
@jericmason I'd like to read the behaviour specs for this, understand what it's used for